lib/datadog/tracing/trace_operation.rb

```diff
@@ -402,6 +404,13 @@ def initialize
           end
         end
 
+        #  Triggered when trace is being propagated between applications or contexts
+        class Propagate < Tracing::Event
```

Comment:
I would like to see this name to reflect the convention established with the other events ("noun-verb") so that (1) its clear what is being propagated and (2) its easier to group events categorically (e.g. by span or by trace)